mp/game/momentum/resource/momentum_english_ref.res

```diff
@@ -329,6 +329,7 @@
         "MOM_Settings_Overlapping_Keys_TT" "When checked, key presses will overlap and null each other out, much like null binds."
         "MOM_Settings_Release_Forward_On_Jump" "Release forward key on jump"
         "MOM_Settings_Release_Forward_On_Jump_TT" "When checked the game will determine the forward key based on movement while on ground and release the determined key when you jump."
+        "MOM_Settings_Draw_Viewmodel" "Show viewmodels"
```

Comment:
Might be good to change to "Show weapon viewmodels" for users that don't really know what a viewmodel is.